src/dparse/ast.d

```diff
@@ -75,6 +75,9 @@ shared static this()
     typeMap[typeid(XorExpression)] = 48;
 }
 
+/// Statement block style
```

Comment:
just repeating the name for something like this isn't really good documentation. I would suggest:

```suggestion
/// Describes which syntax was used in a list of declarations in the containing AST node
```